resources/metadata/mavenBuild.yaml

```diff
@@ -74,6 +74,17 @@ spec:
         default: false
         aliases:
           - name: maven/logSuccessfulMavenTransfers
+      - name: createBOM
```

Comment:
Is this creating any file that we should expose using the `output`?

Reply:
Yes, this step creates bom.xml in target directory.

Reply:
https://github.com/SAP/jenkins-library/blob/e5e0a482fa38c1ea62f7ff814661c17e2f0df508/pkg/piperutils/stepResults.go#L20

https://github.com/SAP/jenkins-library/blob/e5e0a482fa38c1ea62f7ff814661c17e2f0df508/cmd/fortifyExecuteScan.go#L81

Reply:
Let's leave it like this for now.